fixtures/data.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
do we need to load the fixtures from https://github.com/calm/api/blob/master/libraries/stripe/src/stripe-fixtures.json ?

Reply:
i think i addressed this, but let me know if this part is still unclear